src/amp-events.js

```diff
@@ -0,0 +1,23 @@
+/**
+* Enum used to specify custom Amp Events
+*
+* @constant {!Object<string, string>}
+*/
+export const AmpEvents = {
+  VISIBILITY_CHANGE: 'amp:visibilitychange',
+  BIND: {
+    INITIALIZE: 'amp:bind:initialize',
+    RESCAN_TEMPLATE: 'amp:bind:rescan-template',
+    SET_STATE: 'amp:bind:setState',
+  },
```

Comment:
These actually come from the AMP extension `amp-bind`, whose events should probably live in its own file under the `extensions/amp-bind/0.1/` directory. Please either exclude Bind events from this PR or create a new `BindEvents` enum and file in that directory. 

Sorry I didn't mention this in the issue. Nice job being thorough!